web/src/user_card_popover.ts

```diff
@@ -592,8 +604,6 @@ function toggle_sidebar_user_card_popover($target: JQuery): void {
             $(instance.popper).find(".tippy-box").addClass("show-when-reference-hidden");
         },
     );
-
-    current_user_sidebar_user_id = user.user_id;
```

Comment:
I do not get why these code related to `current_user_sidebar_user_id` is removed.